examples/Scikit.ipynb

```diff
@@ -0,0 +1,330 @@
+      "text/plain": [
+       "<Figure size 720x360 with 10 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import time\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "\n",
+    "t0 = time.time()\n",
+    "\n",
+    "# Split data into test and train sets\n",
+    "train_samples = 20000\n",
+    "\n",
+    "X_train, X_test, y_train, y_test = train_test_split(\n",
+    "    X, y, train_size=train_samples, test_size=10000)\n",
+    "\n",
+    "# Define model, lower C value gives higher regulation\n",
+    "clf = LogisticRegression(\n",
+    "    C=0.001, penalty='l2', solver='saga', tol=0.1\n",
+    ")\n",
+    "\n",
+    "# Fit model\n",
+    "clf.fit(X_train, y_train)\n",
+    "\n",
+    "# Score model\n",
+    "score = clf.score(X_test, y_test)\n",
+    "print(\"Test score with L2 penalty: %.4f\" % score)\n",
+    "\n",
+    "# Visualize coefficients\n",
+    "coef = clf.coef_.copy()\n",
+    "max_val = np.abs(coef).max()\n",
+    "\n",
+    "plt.figure(figsize=(10, 5))\n",
+    "for i in range(10):\n",
+    "    p = plt.subplot(2, 5, i + 1)\n",
+    "    p.imshow(coef[i].reshape(28, 28), cmap=plt.cm.RdBu,\n",
+    "             vmin=-max_val, vmax=max_val)\n",
+    "    p.set_xticks(())\n",
+    "    p.set_yticks(())\n",
+    "    p.set_xlabel('Class %i' % i)\n",
+    "plt.suptitle('Classification vector for...')\n",
+    "\n",
+    "run_time = time.time() - t0\n",
+    "print('Example run in %.3f s' % run_time)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52a034e3",
+   "metadata": {},
+   "source": [
+    "## The \"Application\"\n",
+    "\n",
+    "We can use the `xaitk-saliency` high-level API to generate saliency maps that compare to the visualization above.\n",
+    "\n",
+    "Our \"application\" will accept a set of images, a blackbox image classifier, and a saliency generator and will generate saliency maps for each image provided.\n",
+    "It will then combine the heatmaps of each corresponding class and plot them.\n",
+    "This should give us a view of the decision boundaries for each class similar to the one above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "08fb1a3e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from smqtk_classifier import ClassifyImage\n",
+    "from xaitk_saliency import GenerateImageClassifierBlackboxSaliency\n",
+    "from functools import reduce\n",
+    "from math import ceil\n",
+    "\n",
+    "def app(\n",
```

Comment:
I think here it would be nice for the app to display the digit below each saliency map, similar to above for the scikit-learn based visualization.